examples/rescript.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ppx-spice-examples",
+  "name": "spice-ppx-examples",
```

Comment:
nit: need to change the name?

Reply:
This was giving me a warning from the new `rewatch` build system, something about breaking package resolution

Reply:
I get it now. Nevermind, I'll fix these names later.